_posts/2022-08-08-WX_Blog1.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Blog 1"
+author: Wang Xu
+---
+
+The main topics of today's lecture are:
+1. Introduction to Monte Carlo methods.
+    Monte Carlo methods calculate values through random sampling.
+    Implementation is:
+    a. with y = f(x), to estimate Area = Integer of f(x)dx;
+    b. Generate a set of uniformly distributed points;
+    c. Count the points below the curve;
+    d. Area ~ Number of points below the curve / Total number of points generated;
+    The more points generated, the more accurate the results
+
+2. Implementation of Monte Carlo method to calculate pi
+    Procedure as follows:
+    a. Start with counts inside the circle to be 0: i.e. count = 0;
+    b. Generate a random point;
+    c. Determine is the random point lies in the unit circle, by calculating the distance of the point to the origin point;
+    d. if the point is inside the circle, count += 1;
+    e. Repeat steps b-d for as many times as possible;
+    f. Calculate the ratio of points inside the circle / total points;
+
+3. Special command line: jupyter notebook --no-browser
```

Comment:
I think mac users don't need "--no-browser" part, but good catch! I almost forgot this line!